repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilterRouterImpl.java

```diff
@@ -193,7 +193,8 @@ public void route(HttpServletRequestWrapper servletRequest, HttpServletResponse
                             LOG.error("Error reading request content", e);
                             servletResponse.sendError(HttpServletResponse.SC_REQUEST_ENTITY_TOO_LARGE, "Error reading request content");
                         } else {
-                            LOG.error("Connection Refused to {}", location.getUri(), e);
+                            LOG.error("Error communicating with {}", location.getUri());
+                            LOG.trace("", e);
```

Comment:
I think this is what we proposed. It's something that we do in a number of other places as well.

As for why, I couldn't really say. I mean, the idea must be to keep the log clean and separate concerns. The rationale may be that Repose will automatically "recover" so there is no need to inform the user of the nitty-gritty. On the other hand, though, this does seem like a rare error where having more information may be useful.

Which is my long-winded way of saying iunno.